android/build.gradle

```diff
@@ -14,24 +14,24 @@ buildscript {
     }
 }
 
-rootProject.allprojects {
-    repositories {
-        google()
-        mavenCentral()
-    }
-}
-//For Release version
-
-
 // rootProject.allprojects {
 //     repositories {
 //         google()
 //         mavenCentral()
-//         maven {
-//            url  "https://s01.oss.sonatype.org/content/repositories/snapshots/"
-//         }
 //     }
 // }
+//For Release version
+
+
+rootProject.allprojects {
+    repositories {
+        google()
+        mavenCentral()
+        maven {
+           url  "https://s01.oss.sonatype.org/content/repositories/snapshots/"
```

Comment:
What is this URL for ?

Reply:
@chiragphyllo its for snapshot version , for android first we are testing with Snapshot version in android .